main/acle.md

```diff
@@ -1990,8 +1998,8 @@ be found in [[BA]](#BA).
 | [`__ARM_FEATURE_SVE_MATMUL_FP32`](#multiplication-of-32-bit-floating-point-matrices)                                                                    | 32-bit floating-point matrix multiply extension (FEAT_F32MM)                                       | 1           |
 | [`__ARM_FEATURE_SVE_MATMUL_FP64`](#multiplication-of-64-bit-floating-point-matrices)                                                                    | 64-bit floating-point matrix multiply extension (FEAT_F64MM)                                       | 1           |
 | [`__ARM_FEATURE_SVE_MATMUL_INT8`](#multiplication-of-8-bit-integer-matrices)                                                                            | SVE support for the integer matrix multiply extension (FEAT_I8MM)                                  | 1           |
-| [`__ARM_FEATURE_SVE_PREDICATE_OPERATORS`](#scalable-vector-extension-sve)                                                                               | C and C++ operators support fixed-length SVE predicate types                                       | 1           |
-| [`__ARM_FEATURE_SVE_VECTOR_OPERATORS`](#scalable-vector-extension-sve)                                                                                  | C and C++ operators support fixed-length SVE vector types                                          | 1           |
+| [`__ARM_FEATURE_SVE_PREDICATE_OPERATORS`](#scalable-vector-extension-sve)                                                                               | C and C++ operators support, 1 for fixed-length SVE predicate types or 2 for scalable SVE predicate types          | 1           |
```

Comment:
This might be too much detail for this table.  How about something like:

> Level of support for C and C++ operators on SVE vector types

?